src/main/java/edu/pdx/cs/sftp/User.java

```diff
@@ -4,45 +4,51 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-/** User class for SFTP project. Used to gather information about a connection */
+/** Gathers information about a connection initiated by the user with the User class */
 class User {
 
-  /** Field for password string */
-  String password;
-  /** Field for username string */
+  /** Store the username in this variable */
   String username;
-  /** Field for host name string */
+  /** Store the password in this variable */
+  String password;
+  /** Store the hostname in this variable */
   String hostname;
-  /** Field for an IO scanner used for input */
+  /** Store IO scanner in this variable */
   private Scanner scanner = new Scanner(System.in);
 
-  /** Default constructor initializes all fields to null */
+  /** Initializes all fields to null with the default constructor */
```

Comment:
The way it was written before actually follows Oracle's javadoc styleguide (https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#styleguide)
It follows this general pattern: What the method is and what it does.

I'd maybe rewrite the old `/** Default constructor initializes all fields to null */` with `/** Class constructor initializing all fields to null */`

Reply:
Example for javadoc constructor comments provided by Oracle:
```
   /**
    * Class constructor specifying number of objects to create.
    */
   foo(int n) {
     ...
```

Reply:
ok will look into it for the next pull request.